RealmSwift/Migration.swift

```diff
@@ -192,7 +193,7 @@ internal func accessorMigrationBlock(_ migrationBlock: MigrationBlock) -> RLMMig
         }
 
         // run migration
-        migrationBlock(migration: Migration(migration), oldSchemaVersion: oldVersion)
+        migrationBlock(/* migration: */ Migration(migration), /* oldSchemaVersion: */ oldVersion)
```

Comment:
Ok. I'm hoping that eventually lambda variables can be named `migrationBlock(migration:oldSchemaVersion:)` so we get our argument labels back.